src/main/webapp/WEB-INF/views/DesignPage.html

```diff
@@ -0,0 +1,339 @@
+    <!--[if lt IE 9]>
+    <script src="https://oss.maxcdn.com/html5shiv/3.7.3/html5shiv.min.js"></script>
+    <script src="https://oss.maxcdn.com/respond/1.4.2/respond.min.js"></script>
+    <![endif]-->
+</head>
+<body>
+<div class="navbar navbar-default navbar-fixed-top">
+    <div class="container">
+        <div class="navbar-header">
+            <button type="button" class="navbar-toggle" data-toggle="collapse"
+                    data-target="#responsive-menu">
+                <span class="sr-only">Menu</span>
+                <span class="icon-bar"></span>
+                <span class="icon-bar"></span>
+                <span class="icon-bar"></span>
+            </button>
+            <a class="navbar-brand" href="#">Great<i class="fa fa-rocket fa-lg" aria-hidden="true"></i>Start </a>
+        </div>
+        <div class="collapse navbar-collapse" id="responsive-menu">
+            <ul class="nav navbar-nav navbar-left">
+                <li class="active"><a href="#">Home</a></li>
+                <li><a href="#">About</a></li>
+                <li><a href="#">Events</a></li>
+                <li><a href="#">Projects</a></li>
+                <li>
+                    <form role="search" class="navbar-form navbar-left">
+                        <div class="form-group">
+                            <input type="text" placeholder="Search" class="form-control">
+                        </div>
+                    </form>
+                </li>
+            </ul>
+            <ul class="nav navbar-nav navbar-right">
+                <!--<li class="active"><a  class="btn btn-success btn-md" href="#modal-login" data-toggle="modal" >Login</a></li>-->
+                <li class="active"><a href="#modal-login" data-toggle="modal">Login</a></li>
+            </ul>
+        </div>
+    </div>
+</div>
+<div id="headerwrap">
+    <div class="container">
+        <div class="row centered">
+            <div class="col-lg-8 col-lg-offset-2">
+                <h1>GreatStart is awesome!</h1>
+                <h2>Join us</h2>
+            </div>
+        </div>
+    </div>
+</div>
+<div class="container w">
+    <div class="row centered">
+        <div class="col-lg-4 col-md-4 col-sm-4">
+            <i class="fa fa-lightbulb-o fa-3x"></i>
+            <h4>Ideas</h4>
+            <p><b>Just Text Example</b><br>
+                Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et
+                dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip
+                ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu
+                fugiat nulla pariatur.</p>
+        </div>
+        <div class="col-lg-4 col-md-4 col-sm-4">
+            <i class="glyphicon glyphicon glyphicon-grain gi-3x"></i>
+            <h4>Help</h4>
+            <p><b>Just Text Example</b><br>
```

Comment:
Here SonarQube is right. NEVER apply bold font by using tag <b>. This type of usage is already deprecated for a long time. You should do it via CSS, or use other tags as advised by SonarQube